src/main/java/org/refactoringminer/rm1/GitHistoryRefactoringMinerImpl.java

```diff
@@ -723,15 +718,14 @@ private GitHub connectToGitHub() {
 					InputStream input = new FileInputStream("github-oauth.properties");
 					prop.load(input);
 					oAuthToken = prop.getProperty("OAuthToken");
+					if (oAuthToken == null || oAuthToken.isEmpty()) throw new OAuthTokenNotFoundException();
 				}
-				if (oAuthToken != null) {
-					gitHub = GitHub.connectUsingOAuth(oAuthToken);
-					if(gitHub.isCredentialValid()) {
-						logger.info("Connected to GitHub with OAuth token");
-					}
+				gitHub = GitHub.connectUsingOAuth(oAuthToken);
+				if(gitHub.isCredentialValid()) {
+					logger.info("Connected to GitHub with OAuth token");
 				}
 				else {
-					gitHub = GitHub.connect();
```

Comment:
`GitHub.connect()` is an alternative connection way to OAuth. We should not remove it.